lib/datadog/appsec/assets.rb

```diff
@@ -10,6 +10,10 @@ def waf_rules(kind = :recommended)
         read("waf_rules/#{kind}.json")
       end
 
+      def waf_processors
+        read('waf_rules/processors.json')
```

Comment:
Ha our `Appsec::Processor` naming is a bit unfortunate. Let's make sure there's no confusion in our code.